packages/components/toggle-button/src/toggle-button.scss

```diff
@@ -1,134 +1,90 @@
-  --_padding-inline: var(--sl-space-button-ghost-icon-only-md);
+  --_bg-color: transparent;
+  --_bg-mix-color: var(--sl-color-background-secondary-interactive-plain);
+  --_bg-opacity: var(--sl-opacity-interactive-plain-idle);
   --_transition-duration: var(--sl-animation-button-duration);
   --_transition-easing: var(--sl-animation-button-easing);
-  --sl-icon-container-size: var(--sl-size-button-icon-container-icon-only-md);
 
-  background: var(--_background);
-  border: var(--_border-width) solid var(--_border-color);
+  // these are overwritten in toggle group
+  --_border-width: var(--sl-size-borderWidth-action);
+  --_button-border-radius: var(--sl-size-borderRadius-default);
+  --_pill-compensation-start: 0px;
+  --_pill-compensation-end: 0px;
+  --_border-compensation: var(--_toggle-group-border, var(--_border-width));
+
+  background: color-mix(in srgb, var(--_bg-color), var(--_bg-mix-color) calc(100% * var(--_bg-opacity)));
+  border: var(--_border-width) solid var(--sl-color-border-secondary-bold);
   border-radius: var(--_button-border-radius);
-  color: var(--_color);
+  color: var(--sl-color-foreground-secondary-bold);
   cursor: pointer;
   display: inline-flex;
-  font: var(--_font);
   outline-color: transparent;
   transform: var(--_transform);
   vertical-align: middle;
 }
 
 :host(:hover) {
-  --_background: var(--sl-color-button-default-ghost-hover-background);
-  --_border-color: var(--sl-color-button-default-ghost-hover-border);
-  --_color: var(--sl-color-button-default-ghost-hover-foreground);
+  --_bg-opacity: var(--sl-opacity-interactive-plain-hover);
 }
 
 :host(:active) {
-  --_background: var(--sl-color-button-default-ghost-active-background);
-  --_border-color: var(--sl-color-button-default-ghost-active-border);
-  --_color: var(--sl-color-button-default-ghost-active-foreground);
-}
-
-:host([pressed]) {
-  --_background: var(--sl-color-button-default-ghost-selected-idle-background);
-  --_border-color: var(--sl-color-button-default-ghost-selected-idle-border);
-  --_color: var(--sl-color-button-default-ghost-selected-idle-foreground);
-}
-
-:host([pressed]:hover) {
-  --_background: var(--sl-color-button-default-ghost-selected-hover-background);
-  --_border-color: var(--sl-color-button-default-ghost-selected-hover-border);
-  --_color: var(--sl-color-button-default-ghost-selected-hover-foreground);
-}
-
-:host([pressed]:active) {
-  --_background: var(--sl-color-button-default-ghost-selected-active-background);
-  --_border-color: var(--sl-color-button-default-ghost-selected-active-border);
-  --_color: var(--sl-color-button-default-ghost-selected-active-foreground);
-}
-
-:host([disabled]) {
-  --_background: var(--sl-color-button-default-ghost-disabled-background);
-  --_color: var(--sl-color-button-default-ghost-disabled-foreground);
-
-  cursor: default;
-  pointer-events: none;
+  --_bg-opacity: var(--sl-opacity-interactive-plain-active);
 }
 
 :host(:focus-visible) {
   outline: var(--_focus-outline);
```

Comment:
In most refactored components these styles (`outline` and `outline-offset`) are set on `:host` and online the `outline-color` is set in `:focus-visible`.